.github/workflows/build.yml

```diff
@@ -1,23 +1,22 @@
-version: 2.1
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+
 jobs:
-  deploy:
-    docker:
-      - image: google/cloud-sdk@sha256:04dbcfd6e6df1d10420d1de94607f8383cc887a5ca81bcc620d037b8a3c1f95a
+  build:
+    runs-on: ubuntu-latest
     steps:
-      - checkout
+      - uses: google-github-actions/setup-gcloud@v2
+
+      - uses: actions/checkout@v4
```

Comment:
Can we just pin to the latest versions?

e.g. https://github.com/open-telemetry/opentelemetry-go-vanityurls/blob/9e48764dc8753eaf7dd0f4d66c95d085717aac9a/.github/workflows/fossa.yml#L15